Runtime/Triangulator.cs

```diff
@@ -2795,19 +2845,69 @@ static float pseudoAngle(float dx, float dy)
         public readonly float mul(float a, float b) => a * b;
         public readonly float neg(float v) => -v;
         public readonly float2 neg(float2 v) => -v;
+
+        static float cross(float2 a, float2 b) => a.x * b.y - a.y * b.x;
+
+        public readonly bool pointInsideTriangle(float2 p, float2 a, float2 b, float2 c) {
+            float3 Barycentric(float2 a, float2 b, float2 c, float2 p) {
```

Comment:
Ah. I just copied this implementation from my previous PR, which was before you had made the rename. Forgot about it.

Is it ok if I leave it like this? Because after this PR, I'm going to submit another one to remove the barycentric method altogether, since there are much simpler and more numerically robust methods to do a point in triangle test.

Reply:
To reduce the changes in the PR it is good idea to restore the previous convention. Better implementations are welomed in the future PRs!

Reply:
Done